amida-tech/amida-notification-microservice
# [DEVOPS-390] Version bump husky. lint/test on pre-push (previously pr…

To test:
- Create a dummy branch
- Make a test commit. Linting shouldn't run.
- Attempt to push to a remote dummy branch. Linting should run and fail because there are linting errors

Linting errors being addressed in https://github.com/amida-tech/amida-notification-microservice/pull/26
